contracts/price_oracles/AggregateStablePrice3.vy

```diff
@@ -0,0 +1,253 @@
+# @version 0.3.10
+"""
+@title AggregatorStablePrice - aggregator of stablecoin prices for crvUSD
+@author Curve.Fi
+@license Copyright (c) Curve.Fi, 2020-2023 - all rights reserved
+"""
+# Returns price of stablecoin in "dollars" based on multiple redeemable stablecoins
+# Recommended to use 3+ price sources
+
+interface Stableswap:
+    def price_oracle(i: uint256=0) -> uint256: view
```

Comment:
This would not accommodate non-ng pools I guess? Are you expecting all pegkeeper pools to be purely ng pools, where you actually need to provide an index, as opposed to non-ng 2-coin pools which do not need an index?

Reply:
Why is that? Calling `.price_oracle()` will use old method_id

Reply:
Right! My bad.